modules/mongodb/templates/mongo-backup-s3.sh.erb

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+backup_node=<%= @backup_node %>
+backup_dir=<%= @backup_dir %>
+backup_file=mongodump-<%= @fqdn %>.$(date +%F_%T).tar.gz.gpg
+key_fingerprint=<%= @private_gpg_key_fingerprint %>
+s3_bucket=<%= @s3_bucket %>
+
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+echo " STARTING BACKUP..................."
+
+mongodump -h $backup_node -o $backup_dir
+
+echo " COMPRESSING AND ENCRYPTING................."
+
+tar cvz - $backup_dir | gpg -e -o $backup_dir/$backup_file -r $key_fingerprint
+
+export AWS_ACCESS_KEY_ID=<%= @aws_access_key_id %>
+export AWS_SECRET_ACCESS_KEY=<%= @ws_secret_access_key %>
```

Comment:
Yes, I think if we change the permissions it shouldn't live in `/usr/local/bin`. Happy to just move the script, too.